File: network-client/src/main/java/com/mgtriffid/academ/network/client/StatesReceived.java
package com.mgtriffid.academ.network.client;

import com.mgtriffid.academ.logic.WorldState;
import org.apache.commons.lang3.tuple.Pair;

import java.util.List;

/**
 * Created by mgtriffid on 17.04.17.
 */
public class StatesReceived {
    private final List<Pair<Integer, WorldState>> states;

    public StatesReceived(List<Pair<Integer, WorldState>> states) {
        this.states = states;
    }

    public List<Pair<Integer, WorldState>> states() {
        return states;
    }
}

File: network-client/src/main/java/com/mgtriffid/academ/network/client/StatesReceived.kt
package com.mgtriffid.academ.network.client

import com.mgtriffid.academ.logic.WorldState

/**
 * Created by mgtriffid on 17.04.17.
 */
class StatesReceived(private val states: List<Pair<Int, WorldState>>) {

    fun states(): List<Pair<Int, WorldState>> {
        return states
    }
}
